application/cms/views.py

```diff
@@ -519,7 +533,10 @@ def _send_to_review(topic_slug, subtopic_slug, measure_slug, version):  # noqa:
                 "error_summary": get_error_summary_data(
                     title="Cannot submit for review.",
                     forms=[measure_version_form, data_source_form, data_source_2_form],
+                    additional_error_messages=non_form_error_messages,
```

Comment:
Doing that makes it much more of a faff to test though I think.

Reply:
Also, the `errors` key might not exist here if the forms validated OK, so would need a bunch of logic to check for the `errors` key and add it if needs be.  Which isn't insurmountable at all, but doesn't feel ny better than what's here maybe.

Reply:
Hmm, maybe. Doing it would mean that we have to test that the error is rendered onto the page, which arguably is the more important thing to do than assert that `list.extend` works - essentially what this new test does? If we leave this as-is, we should probably add/update a test that validates we get errors as expected for missing date files.

Reply:
🤦‍♂️ No errors key, of course. Probably better as you've got here than my half-baked idea, then. 👍